server/database_operations.js

```diff
@@ -0,0 +1,25 @@
+const { Pool } = require("db");
+const config = require("./config");
```

Comment:
I think in here you need

```js
const { config } = require("./config");
```

as you are exported as a named export (_not default export_)